src/System.Console/src/System/IO/SyncTextReader.cs

```diff
@@ -11,7 +11,6 @@ namespace System.IO
     internal sealed class SyncTextReader : TextReader
     {
         private readonly TextReader _in;
-        private readonly object _methodLock = new object();
```

Comment:
@Maxwe11 Thanks for the feedback. I agree with @stephentoub and would prefer to use lock(this). It gives an additional benefit of being consistent with the desktop code as both code basis now have same implementation.